topics/icrc_34_get_delegation.md

```diff
@@ -0,0 +1,192 @@
+# ICRC-34: Get Delegation
+
+[![Status Badge](https://img.shields.io/badge/STATUS-DRAFT-ffcc00.svg)](https://github.com/orgs/dfinity/projects/31)
+[![Extension Badge](https://img.shields.io/badge/EXTENDS-ICRC--25-ffcc222.svg)](./icrc_25_signer_interaction_standard.md)
+[![Standard Issue](https://img.shields.io/badge/ISSUE-ICRC--34-blue?logo=github)](https://github.com/dfinity/wg-identity-authentication/issues/115)
+
+<!-- TOC -->
+* [ICRC-34: Get Delegation](#icrc-34-get-delegation)
+  * [Summary](#summary)
+  * [Method](#method)
+  * [Scope (according to the ICRC-25 standard)](#scope-according-to-the-icrc-25-standard)
+    * [Example RPC Permission Request](#example-rpc-permission-request)
+  * [Requests](#requests)
+    * [Example RPC Get Delegation Request](#example-rpc-get-delegation-request)
+    * [Example RPC Get Delegation Response (wallet delegation)](#example-rpc-get-delegation-response-wallet-delegation)
+    * [Example RPC Get Delegation Response (app-specific delegation)](#example-rpc-get-delegation-response-app-specific-delegation)
+  * [Message Processing](#message-processing)
+  * [Errors](#errors)
+<!-- TOC -->
+
+## Summary
+
+When a relying party wants to remove wallet approval prompts and sign on behalf of its users, it uses the
```

Comment:
See comment above. I think we should talk here about removing interactions with the signer.